test/client-test.js

```diff
@@ -442,13 +442,16 @@ describe('SOAP Client', function() {
         var data = {
           Request: {
             $attributes: {
-              $xsiType: '{xmlnsTy}Ty'
+              $xsiType: {
+                "type": "Ty",
+                "xmlns": "xmlnsTy"
```

Comment:
@raymondfeng I will check for consistency with other features and get back to you